src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/model/FlowPathLatency.java

```diff
@@ -22,4 +22,8 @@
 
     private long timestamp;
```

Comment:
What time units stored in timestamp? Can we have some unit reference in field name? Or better use `java.time.Instant` instead of `long`?